### PR TITLE
fix: update dockerfile to remove python reqs

### PR DESCRIPTION
**Pull request by JeremyWesthead**
I imagine this is 90+% of what vulnerability scanners were picking up on

### CODE REVIEW

**Review by mcolpus — APPROVED** [review]
So is it just not used at all now?

File: environments/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:latest
```
Comment (bobturneruk) [inline environments/Dockerfile:1]: I think we should pin a version, in case we get unexpected behaviour in future.

Comment (JeremyWesthead) [reply environments/Dockerfile:1]: We could, but the only unexpected behaviour ever seen here was around python versions being bumped. This has essentially 0 dependencies, so should be fine

Comment (bobturneruk) [reply environments/Dockerfile:1]: Fair enough. I felt I had to check. 😸

**Pull request merged**